scripts/add_electricity.py

```diff
@@ -550,13 +559,20 @@ def attach_hydro(n, costs, ppl):
 
     if "hydro" in carriers and not hydro.empty:
         hydro_max_hours = c.get("hydro_max_hours")
-        hydro_stats = pd.read_csv(
-            snakemake.input.hydro_capacities, comment="#", na_values=["-"], index_col=0
+        hydro_stats = (
+            pd.read_csv(
+                snakemake.input.hydro_capacities,
+                comment="#",
+                na_values=["-"],
+                index_col=0,
+            )
+            .groupby("Country")
+            .sum()
         )
         e_target = hydro_stats["E_store[TWh]"].clip(lower=0.2) * 1e6
```

Comment:
What is the source of 0.2 TWh constant here? Would it be possible to add some explanation on that?

Reply:
PyPSA-Eur XD they used this number. It means a small-enough number, we can drop it to 0.1 in case.
The aim is to avoid too small units.
The assumption here is that hydro units below 0.2TWh do not exists (200MWh)

Reply:
My feeling is that it could probably make sense to add a comment on this. What do you think?

Reply:
I think it may not be needed. If someone investigates the code, it may be easy to understand.
Otherwise, the risk is adding too many comments otherwise.